dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -0,0 +1,47 @@
+# == Schema Information
+#
+# Table name: circuit_playground_discount_applications
+#
+#  id                                  :integer          not null, primary key
+#  user_id                             :integer          not null
+#  unit_6_intention                    :integer
+#  has_confirmed_school                :boolean          default(FALSE), not null
+#  full_discount                       :boolean
```

Comment:
Not sure if it's an abuse/bad practice to have a boolean that allows null. This particular field will have no meaning until we actually have a discount_code, so we could just have it always be false (or even true) until that point, but allowing it to be nil made more sense to me.

Reply:
Not sure what others think, but I find the tri-state of null/false/true to be really useful, as long as the semantics of the null value are well-defined, as they seem to be here (irrelevant/unknown).